interface/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM phusion/baseimage:0.9.22
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+# libfontconfig is needed for phantomjs, which is apparently installed in
+# our ubuntu machines, but required with this phusion/baseimage:0.9.22
+RUN apt-get update && apt-get install -y \
+  nodejs-legacy \
+  npm \
+  git \
+  libfontconfig
+
+# Install global node packages.
+RUN npm -g install grunt-cli karma-cli bower
+
+# Clean up APT when done.
+RUN apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Copy code to server directory
+COPY . .
```

Comment:
Actually we don't need to worry about it at all for this Dockerfile, since it only includes the files in this directory (so it will not include any of the files in the parent directory, or its subdirectories, like the config.py file). In fact, I don't think we will need any secrets for the interface portion. However, I will be careful to make sure and ignore it in the Dockerfile for the backend.